packages/code-du-travail-frontend/src/search/Search.js

```diff
@@ -224,6 +224,10 @@ export default withRouter(Search);
 
 const { animations, box, breakpoints, colors, spacing, fonts } = theme;
 
+const SearchSection = styled(Section)`
+  box-shadow: 0 10px 10px -10px ${colors.lightGrey};
```

Comment:
could we use `theme.box.shadow` ?

Reply:
I'll give it a try

Reply:
<img width="1180" alt="Screenshot 2019-05-14 at 13 29 45" src="https://user-images.githubusercontent.com/705453/57694572-5eb4a580-764c-11e9-8169-c053456ee1e6.png">
Slightly too strong isn't it ?

Reply:
Previous: 
<img width="1185" alt="Screenshot 2019-05-14 at 13 30 58" src="https://user-images.githubusercontent.com/705453/57694630-8a379000-764c-11e9-9648-5f0b8b0d086c.png">

Reply:
But we could make the light one be the default !